models/topic.go

```diff
@@ -26,6 +26,22 @@ type Topic struct {
 	SubtopicIds []string    `bson:"subtopics_ids,omitempty"  json:"-"`
 }
 
+type TopicWrite struct {
```

Comment:
it might be best to keep these inside of the steps file for the time being, given they are only used for test purposes

Reply:
No, if the TopicResponse and / or Topic struct ever change then keeping the new structs close would be preferred.
Added comments to the new structs.